docs/content.zh/docs/dev/datastream/scala_api_extensions.md

```diff
@@ -184,22 +176,21 @@ data1.join(data2).
 
 
 
-For more information on the semantics of each method, please refer to the [DataStream]({{< ref "docs/dev/datastream/overview" >}}) API documentation.
+有关每个方法语义的更多信息, 请参考 [DataStream]({{< ref "docs/dev/datastream/overview" >}}) API 文档。
 
-To use this extension exclusively, you can add the following `import`:
+要单独使用此扩展，您可以添加以下引入：
 
 {{< highlight scala >}}
 import org.apache.flink.api.scala.extensions.acceptPartialFunctions
 {{< /highlight >}}
 
-for the DataSet extensions and
+用于DataSet扩展
 
 {{< highlight scala >}}
 import org.apache.flink.streaming.api.scala.extensions.acceptPartialFunctions
 {{< /highlight >}}
 
-The following snippet shows a minimal example of how to use these extension
-methods together (with the DataSet API):
+下面的代码片段展示了如何一起使用这些扩展方法(以及 DataSet API)的最小示例:
```

Comment:
```suggestion
下面的代码片段展示了如何一起使用这些扩展方法（以及 DataSet API）的最小示例:
```

Reply:
hi @RocMarshal Thank you for your suggestion, I have revised it again, is there anything else I can do？